README.md

```diff
@@ -575,7 +575,9 @@ mv runTauIdMVA.py RecoTauTag/RecoTau/python/tools/
 #Minor modification to DataFormat
 git cms-addpkg DataFormats/Math/
 
-#Add ```<class name="edm::Wrapper<ROOT::Math::SMatrix<double,2,2,ROOT::Math::MatRepSym<double,2> > >"/>``` to DataFormats/Math/src/classes_def.xml
+#Add string to class_def.xml
```

Comment:
being picky, this could mention which `class_def.xml` has to be changed (ie `DataFormats/Math/src/classes_def.xml)`